dist/index.js

```diff
@@ -0,0 +1,166 @@
+const addTasksBtn = document.getElementById('add-task-btn'); // call button for add tasks
+const deskTaskInput = document.getElementById('description-task'); // the tasks input
+const todoWrapper = document.querySelector('.todos-wrapper'); // the tasks section where they will be displayed
+
+let tasks; // this is an empty array of our taks
+/* eslint-disable */
+!localStorage.tasks ? tasks = [] : tasks = JSON.parse(localStorage.getItem('tasks')); // check the local storage of any tasks
+/* eslint-anable */
+
+let todoItemElems = [];
+
+function Task(description) { // this is a constructor for each task
+  this.description = description; // this comes from input
+  this.completed = false; // by default the task will be not completed
+  this.index = tasks.length + 1; // the index of inputed task
+}
+/* eslint-disable */
```

Comment:
- [ ] As per the learning objective, We have to use proper ES6 syntax. Kindly try to use the arrow function as per ES6 guidelines.

- [ ] Kindly try to delete the comments from the file.